config/config_test.go

```diff
@@ -354,7 +354,7 @@ request_validation:
     ipv4_private_networks: ["1.1.1.0/24"]
     ipv6_private_networks: ["1111::/16", "2222::/16"]
 generate_bid_id: true
-enable_legacy_auction: false
+enable_legacy_auction: true
```

Comment:
For the `TestFullConfig` test, can we leave its value as `false` just for the sake of it being different to the default value of `true`